app/scripts/lib/setupSentry.js

```diff
@@ -30,6 +30,53 @@ export const SENTRY_BACKGROUND_STATE = {
   AccountTracker: {
     currentBlockGasLimit: true,
   },
+  PermissionLogController: {
+    // double-check if contains PID?
+    permissionHistory: true,
+    // double-check if contains PID?
+    permissionActivityLog: true,
+  },
+  AnnouncementController: {
+    announcements: true,
+  },
+  GasFeeController: {
+    gasFeeEstimates: true,
+    estimatedGasFeeTimeBounds: true,
+    gasEstimateType: true,
+  },
+  TokenListController: {
+    // double-check if contains PID?
+    tokenList: true,
+    tokensChainsCache: true,
+    preventPollingOnNetworkRestart: true,
+  },
+  TokensController: {
+    // double-check if contains PID?
+    tokens: true,
+    // double-check if contains PID?
+    ignoredTokens: true,
+    // double-check if contains PID?
+    detectedTokens: true,
+    // double-check if contains PID?
+    allTokens: true,
+    // double-check if contains PID?
+    allIgnoredTokens: true,
+    // double-check if contains PID?
+    allDetectedTokens: true,
+  },
+  SmartTransactionsController: {
+    smartTransactions: false,
+    userOptIn: true,
+    liveness: true,
+    fees: {
+      approvalTxFees: true,
+      tradeTxFees: true,
+    },
+  },
+  NotificationController: {
+    // double-check if contains PID?
+    notifications: true,
```

Comment:
This is part of snaps, I'm not sure exactly what it contains. Maybe best to exclude it for now, especially since it won't shed light on v10.35.0 issues on stable anyway (no snaps there).

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/b3330828235d61e68ab356e3e859a30d3c44af6f